spec/rails_helper.rb

```diff
@@ -60,9 +60,8 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+  
+  # Include FactoryBot's configuraton to write cleaner and simpler codes
+  config.include FactoryBot::Syntax::Methods
```

Comment:
Nice catch, nested `Rspec.configure`'s doesn't seem to break anything, but this is much nicer.